### ISSUE TITLE
[BottomNavigation] Primary color theme does not pass contrast requirements

### ISSUE THREAD

**Issue by romoore** [material-components/material-components-ios#7693]
The Bottom Navigation primary theme with the default color scheme values uses color combinations for selected and unselected items that do not pass the minimum contrast ratios for WCAG 2.1 AA levels. Specifically the ratio is 4.4:1 (as I've seen it) and should be 4.5:1 (minimum).

## Selected Items
```
Foreground: #dbbcff
Background: #6e3ae7
Ratio: 3.70
```

## Unselected Items
```
Foreground: #d1c0f7
Background: #5828e6
Ratio: 4.40
```

<img width="520" alt="testPrimaryThemeWithDefaultContainerScheme_11_2@2x" src="https://user-images.githubusercontent.com/1753199/60026774-3916b500-966a-11e9-903f-22d71037a480.png">
